src/main/java/com/karankumar/booksapi/repository/BookRepository.java

```diff
@@ -32,4 +33,8 @@
     Book findBookByIsbn13(String isbn13);
   
     Book findByTitleIgnoreCase(String title);
+    
+//    @Query("SELECT b FROM Publisher_book pb join pb.books b")
+    @Query("SELECT DISTINCT b FROM Book b LEFT JOIN FETCH b.authors")
```

Comment:
@knjk04 I will need up update this query right?

In SQL terms it should be something like:

`SELECT * FROM Book b LEFT JOIN Publisher_book pb on b.id = pb.book_id LEFT JOIN Publisher p on p.id = pb.publisher_id WHERE p.name = "{NAME}"`

Reply:
Yes, that looks about right. Try adding a @DataJpa test as well to test it works as intended